Lib/defconQt/glyphView.py

```diff
@@ -1,14 +1,16 @@
 from enum import Enum
 from math import copysign
+from functools import partial
 import pickle
 from defcon import Anchor, Component
 from defconQt import icons_db
 from defconQt.objects.defcon import TContour, TGlyph
 from defconQt.pens.copySelectionPen import CopySelectionPen
 from defconQt.util import platformSpecific
+from defconQt import icons_db
```

Comment:
Have you rebased on top of master? It seems you merged the latest changes in the tree in your commits.